src/tools/dart-test.md

```diff
@@ -0,0 +1,137 @@
+---
+title: dart test
+description: Command-line tool for testing Dart projects.
+toc: false
+---
+
+The `dart test` command runs tests that
+rely on the [`test` package][] and
+are under the `test` directory of the current Dart project. ***[PENDING: check]***
+For information on writing tests, see the 
+[testing documentation][].
+If you're working on Flutter code, then use the `flutter test` command instead,
+as described in [Testing Flutter apps][].
+
+[testing documentation]: /guides/testing
+[`test` package]: {{site.pub-pkg}}/test
+[Testing Flutter apps]: {{site.flutter}}/docs/testing
+
+{% include tools/dart-tool-note.md %}
+
+Here's an example of using `dart test` to run all tests
+that are under the current project's `test` directory:
+
+```terminal
+$ cd my_app
+$ dart test
+```
+
+To run a subset of tests, use the `--name` (`-n`),
```

Comment:
You can also pass file paths `dart test test/some_test.dart`, which is probably worth mentioning?